src/index.ts

```diff
@@ -81,6 +81,7 @@ import featureFilter, {isExpressionFilter} from './feature_filter';
 
 import convertFilter from './feature_filter/convert';
 import Color from './util/color';
+import {Projection} from './util/projection';
```

Comment:
Others are "default export"ed, to align with other files I would consider doing the same here, and maybe afterwards do a PR to change things to named exports.